cmake/Packaging.cmake

```diff
@@ -139,6 +138,48 @@ ly_install(FILES ${_cmake_package_dest}
     COMPONENT ${CMAKE_INSTALL_DEFAULT_COMPONENT_NAME}
 )
 
+# Do inital check on workspace path in the event that LY_3RDPARTY_PATH is misconfigured
+file(REAL_PATH "${CMAKE_CURRENT_SOURCE_DIR}/.." _root_path)
+if (EXISTS "${_root_path}/3rdParty/packages")
+    set(CPACK_LY_3P_PACKAGE_DIRECTORY "${_root_path}/3rdParty/packages")
+else()
+    set(CPACK_LY_3P_PACKAGE_DIRECTORY ${LY_3RDPARTY_PATH}/packages)
+endif()
+
+# Scan the engine and 3rd Party folders for licenses
+file(TO_NATIVE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/python/python.cmd" _python_cmd)
+file(TO_NATIVE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/scripts/license_scanner" _license_script_path)
+
+set(_license_script "${_license_script_path}/license_scanner.py")
+set(_license_config "${_license_script_path}/scanner_config.json")
+
+set(_license_scan_path "${CMAKE_CURRENT_SOURCE_DIR}" "${CPACK_LY_3P_PACKAGE_DIRECTORY}")
+set(CPACK_3P_LICENSE_FILE "${CPACK_BINARY_DIR}/NOTICES.txt")
+set(CPACK_3P_MANIFEST_FILE "${CPACK_BINARY_DIR}/SPDX-License.json")
+
+set(_license_command
+    ${_python_cmd} -s
+    -u ${_license_script}
+    --config-file ${_license_config}
+    --license-file-path ${CPACK_3P_LICENSE_FILE}
+    --package-file-path ${CPACK_3P_MANIFEST_FILE}
+)
+
+message(STATUS "Scanning for license files in ${_license_scan_path}")
+execute_process(
+    COMMAND ${_license_command} --scan-path ${_license_scan_path}
+    RESULT_VARIABLE _license_result
+    ERROR_VARIABLE _license_errors
+    OUTPUT_VARIABLE _license_output
+    ECHO_OUTPUT_VARIABLE
+)
+
+if(NOT ${_license_result} EQUAL 0)
+    message(FATAL_ERROR "An error occurred during license scan.  ${_license_errors}")
+else()
+    set(CPACK_RESOURCE_FILE_LICENSE ${CPACK_3P_LICENSE_FILE})
```

Comment:
this variable is not being used on the cpack side

Reply:
Sure, I was told that this was going to get used for the installer, but happy to remove it

Reply:
If the installer needs to include this license, we need to inject it somehow. Is there an issue to follow up on doing that?

Reply:
I'll dig in and make an issue if it's really needed